### PR TITLE
fix config as global variable

### LINKED ISSUE #193: Two configFiles in the same webpack build is not working
### Issue description

When I have 2 entries in my `webpack.config.js`, with a `configFilePath` and a particular bootstrap-loader configuration file, both entries compiles with the first configuration.
### Where can I see the bug?

Repository to expose the bug: https://github.com/navarroaxel/webpack-bootstrap. The repository has 2 webpacks builds to show the differences.
### Versions:

`node` 6.9.1
`webpack` 1.13.2
`bootstrap-loader` 1.2.1

is it a bad config? or is it a bug?``

### PR DESCRIPTION
**Pull request by navarroaxel**
fix #193 
Now, the config is not a global variable and it's independent for each entry in the webpack config.

---
This change is [Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/194)

### CODE REVIEW

**Comment by navarroaxel** [comment]
I don't know how to make unit tests about this, but I can add this [repository](https://github.com/navarroaxel/webpack-bootstrap) to the examples folder to show how to use this.

**Comment by justin808** [comment]
@Judahmeek @AlexKVal I need to get your input on this one.

**Comment by Judahmeek** [comment]
Looks pretty good. Interesting use case. The only improvement I can think of would be creating a constant so that you didn't have to run `JSON.stringify(config)` twice in `bootstrap.loader.js`. Minor detail though. Nice work!

**Comment by navarroaxel** [comment]
Hey @Judahmeek, I have changed the commit to run ``JSON.stringify(config)`` only once.
Also this was my first contribution to the webpack's ecosystem.

**Comment by AlexKVal** [comment]
:shipit:

**Comment by justin808** [comment]
@navarroaxel @Judahmeek can I get confirmation that all the examples are working? Should we add an example that demonstrates the use case this solves?

**Comment by AlexKVal** [comment]
> can I get confirmation that all the examples are working?

Ran all examples

#### basic
- npm run bs3
- npm run bs4

- npm run bs3:prod
- npm run bs4:prod

- npm run bs:no-config

#### css-modules
- npm run bs3
- npm run bs4

- npm run bs3:prod
- npm run bs4:prod

**Everything works fine** 👍 

> Should we add an example that demonstrates the use case this solves?

This upon you to decide.

If @navarroaxel adds an additional example it would be really nice.

**Comment by navarroaxel** [comment]
@justin808 I can add the example after the merge of this pull request.

**Comment by justin808** [comment]
@navarroaxel Please add one now, and we'll try it as part of our final checks before pushing a new version. We've been burned in the past by pushing something too quickly.

**Comment by navarroaxel** [comment]
@justin808 I have added an example: the webpack's config has two entries.

**Comment by navarroaxel** [comment]
Hey @justin808 @AlexKVal, do you have some news about this pull request?

**Comment by justin808** [comment]
@navarroaxel Sorry about that! I didn't get the notification. @AlexKVal Can you take a peek at the new example and if you give me the OK, I'll push a new release!

File: examples/multiple-entries/package.json [hunk 1]
```diff
@@ -0,0 +1,64 @@
+{
+  "name": "bootstrap-loader-example",
+  "version": "1.0.0",
+  "description": "Example usage of bootstrap-loader with multiple entries wit",
```
Comment (AlexKVal) [inline examples/multiple-entries/package.json]: wit ?

**Comment by justin808** [comment]
@AlexKVal Reviewable makes it OK to rebase/squash, etc.

**Comment by AlexKVal** [comment]
> @AlexKVal Reviewable makes it OK to rebase/squash, etc.

Not every oss project uses it.
My piece of advise is generalized.

**Comment by justin808** [comment]
@navarroaxel Please fix the items suggested by @AlexKVal and notify me. I'll release at that point.

**Comment by navarroaxel** [comment]
@justin808 I changed the nodejs version to v6.x on `.travic.yml` because the tests are failing on v7.

**Comment by justin808** [comment]
Thanks @navarroaxel. I'm going to merge this tonight or tomorrow morning.

**Comment by justin808** [comment]
@navarroaxel AWESOME! 👍 🎉

**Comment by justin808** [comment]
Shipped in 1.3.0. The change to 2.x (master) is not completed. It can be found at #195.

**Pull request merged**